### ISSUE TITLE
why not optimize the parameters in LPIPS?

### ISSUE THREAD

**Issue by fnzhan** [CompVis/taming-transformers#125]
There are extra convolution layers (self.lin0, self.lin1, ..., self.lin4) in LPIPS, but why the requires_grad of them are set as False? https://github.com/CompVis/taming-transformers/blob/31216490efe8ae3604efbf9f1531ff5c70bd446a/taming/modules/losses/lpips.py#L25

**Comment by neverix** [comment]
1. LPIPS should be frozen because it is used for loss calculations, and if it was dynamic the model could zero the weights out
2. The extra layers are included in the checkpoint LPIPS is initialized from. The reason it's confusing is that they are actually part of the model

**Comment by fnzhan** [comment]
> 

Thanks for your explanation.

**Comment by fnzhan** [comment]
> 

Thanks for your explanation.
